notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,572 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Style Transfer with OpenVINO™\n",
+    "\n",
+    "This notebook demonstrates style transfer with OpenVINO, using the [Style Transfer Models](https://docs.openvino.ai/2021.3/style_transfer_models_public.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/), specifically [fast-neural-style-mosaic-onnx](https://docs.openvino.ai/2021.3/omz_models_model_fast_neural_style_mosaic_onnx.html) model is one of the style transfer models designed to mix the content of an image with the style of another image. The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/abs/1607.08022). Original ONNX models are provided in the [repository](https://github.com/onnx/models). Final part of this notebook shows live inference results from a webcam. Additionally, you can also upload a video file.\n",
```

Comment:
This link here is outdated (2021.3 version). It's better to link to Open Model Zoo repo model instead.